docs/rsdl/rapid-rsdl-paths.md

```diff
@@ -0,0 +1,103 @@
+---
+id: rsdl-paths
+title: RAPID Paths
+---
+
+# Paths in a RAPID Service
+
+The set of valid paths for a RAPID service is the result of applying common path conventions to the model, as described below. You can optionally specify an subset of these paths that are supported by your service using [explicit paths](rapid-rsdl-intro.md#explicit-paths) in your RSDL.
+
+- [Service Root](#service-root)
+- [Key Segment](#key-segment)
+- [Structured Properties](#structured-properties)
+- [Nested Operations](#nested-operations)
+
+## Service Root
+
+The [service](./rapid-rsdl-intro.md#defining-a-service) element in RSDL defines the segments that are available at the service root:
+
+```rsdl
+service {
+  company: Company
+}
+```
+
+## | supported path |
```

Comment:
This looks weird. And it should rather be a subsection than a section on the same level.
```suggestion
### Supported Path
```